docker-compose.yml

```diff
@@ -1,18 +1,36 @@
 version: "3"
 services:
-  website:
-    image: openstates/new-openstates.org
-    volumes:
-      - .:/opt/openstates/openstates.org/
-    entrypoint: ['/bin/bash']
-
   database:
     # With this service up, your host can use a database URL of
     # `postgres://openstates@localhost:5433/openstates`
-    image: mdillon/postgis:9.6
+    image: mdillon/postgis:9.4-alpine
     ports:
       # Avoid conflict with the host's Postgres port
       - "5433:5432"
     environment:
       - POSTGRES_DB=openstates
       - POSTGRES_USER=openstates
+  website:
+    build:
+      context: .
+      dockerfile: python.Dockerfile
+    command: bash ./scripts/docker_python.sh
+    environment:
+      - DATABASE_URL=postgis://openstates@database:5432/openstates
+    ports:
+      - "8000:8000"
+    depends_on:
+      - database
+    volumes:
+      - .:/opt/openstates
+  frontend:
+    build:
+      context: .
+      dockerfile: node.Dockerfile
```

Comment:
npm is just used for building assets

Reply:
okay

Reply:
On second thought, in the issue it was mentioned that the dockerisation setup was required so that same environment could be setup for local distribution thus avoiding environment based issues. Now I assume during the development process, changes will be made to the javascript files and since a watcher script is being used to monitor any changes in the files and update the build automatically, it became vital that this process be run as an individual service.

However, if this is not the case and the setup will be used to distribute code on the servers as well then I don't think this will be necessary and this could be added in the build process itself.